src/js/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import App from './App';
+import App from './App/App';
```

Comment:
Make index files for each component please. So you can import like 
`import App from './App`